bundles/org.openhab.binding.network/src/main/feature/feature.xml

```diff
@@ -5,6 +5,10 @@
     <feature name="openhab-binding-network" description="Network Binding" version="${project.version}">
         <feature>openhab-runtime-base</feature>
         <feature>openhab-core-model-script</feature>
+        <feature prerequisite="true">wrap</feature>
+        <bundle dependency="true">wrap:mvn:fr.bmartel/jspeedtest/1.32.1$Bundle-Name=Speed%20Test%20library%20for%20Java&amp;Bundle-SymbolicName=fr.bmartel.jspeedtest&amp;Bundle-Version=1.32.1</bundle>
+	<bundle dependency="true">wrap:mvn:fr.bmartel/http-endec/1.04$Bundle-Name=http-endec&amp;Bundle-SymbolicName=fr.bmartel.http-endec&amp;Bundle-Version=1.04</bundle>
         <bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.binding.network/${project.version}</bundle>
```

Comment:
Somehow you are still missing a dependency.
`<bundle dependency="true">mvn:commons-net/commons-net/3.6</bundle>`